### PR TITLE
Adding forms

### PR DESCRIPTION
**Pull request by rolandfbk**
I created forms to perform the following functions:

- Creates a Post on behalf of the current_user (a method that I created in the ApplicationController).

- Creates a comment on behalf of the current_user (a method that I created in the ApplicationController).

I also allowed users to add likes to Posts.

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -1,2 +1,17 @@
 class CommentsController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
```
Comment (KDlamini) [inline app/controllers/comments_controller.rb:7]: Kindly create the `new` method that initiates a new comment as required by rails conventions for controllers.

File: config/routes.rb [hunk 2]
```diff
@@ -5,6 +5,12 @@
   root "users#index"
 
   resources :users, only: %i[index show] do
-    resources :posts, only: %i[index show]
+    resources :posts, only: %i[index new create show] do
+    end
+  end
+
+  resources :posts do
+    resources :comments, only: %i[create]
```
Comment (KDlamini) [inline config/routes.rb]: Please include the `new` route as well.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @rolandfbk 🙋‍♀️,  

Great work so far! 💯 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 
✔️  Good PR title and description
✔️  Correct use of Gitflow 
✔️  Commit messages are meaningful
✔️  No linter errors

## Required Changes ♻️

**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
Hi @rolandfbk, 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**